ebs-janitor/config.go

```diff
@@ -20,6 +20,9 @@ type config struct {
 
 // Validate makes sure that the config makes sense
 func (c *config) Validate() error {
+	if len(c.Region) == 0 {
```

Comment:
nit: I think if you do just `c.Region==""` should be the same, I think it doesn't count the space characters (not sure)